sdk/helper/pluginutil/run_config.go

```diff
@@ -20,6 +20,15 @@ import (
 	"github.com/hashicorp/vault/sdk/helper/pluginruntimeutil"
 )
 
+const (
+	// Labels for plugin container ownership
+	labelVaultPID           = "com.hashicorp.vault.pid"
+	labelVaultClusterID     = "com.hashicorp.vault.clusterID"
+	labelVaultPluginName    = "com.hashicorp.vault.pluginName"
+	labelVaultPluginVersion = "com.hashicorp.vault.pluginVersion"
+	labelVaultPluginType    = "com.hashicorp.vault.pluginType"
```

Comment:
I'm wondering if these should be all lowercased, e.g. "plugin.name" instead of "pluginName". The [OCI image spec](https://github.com/opencontainers/image-spec/blob/main/annotations.md) doesn't mention case, but [Docker seems to recommend](https://docs.docker.com/config/labels-custom-metadata/) all lowercase.

Reply:
Yeah all lower case SGTM 👍